master.cfg

```diff
@@ -0,0 +1,285 @@
+              branch='master',
+              pollinterval=300),
+]
+
+####### BUILDSLAVES
+
+c['slavePortnum'] = config['SLAVE_PORT']
+
+# The 'slaves' list defines the set of recognized buildslaves. Each element is
+# a BuildSlave object, specifying a unique slave name and password.
+c['slaves'] = [BuildSlave(s.name(), s.password()) for s in slaves.GetSlaves()]
+
+####### BUILDERS
+
+# The 'builders' list defines the Builders, which tell Buildbot how to perform
+# a build: what steps, and which slaves can execute them.  Note that any
+# particular build will only take place on one slave.
+
+# The main factory, to do fetch via git, autoreconf, configure, make & make
+# check.
+factory = BuildFactory()
+# check out the source
+factory.addStep(Git(repourl=("%s%s" % (config['BASEURL'], config['REPOSTUB']))))
+factory.addStep(ShellCommand(
+    command=["qmake"],
+    name="qmake",
+    haltOnFailure=True,
+    flunkOnFailure=True,
+))
+factory.addStep(Compile(
+    name='make',
+    description='building',
+    descriptionDone='build',
+    command=['make'],
+))
+factory.addStep(Compile(
+    name='unit tests',
+    description='testing',
+    descriptionDone='test',
+    command=['./unittest.sh'],
+    logfiles=config['LOGFILES'],
+    lazylogfiles=True,
+))
+
+# cpplint factory
+cpp_lint_command = (
+    'cpplint.py %s $(find ./ -name "*.h" -or -name "*.cpp" | xargs)' %
+    config['CPP_LINT_ARGS'])
+cpp_lint_factory = BuildFactory()
+cpp_lint_factory.addStep(Git(repourl=("%s%s" % (config['BASEURL'], config['REPOSTUB']))))
+cpp_lint_factory.addStep(ShellCommand(
+   command=cpp_lint_command,
+   haltOnFailure=False,
+   flunkOnFailure=False,
+   warnOnWarnings=True,
+   warnOnFailure=True,
+   description='linting',
+   descriptionDone='lint',
+))
+
+# jslint factory
+js_lint_command = (
+    'gjslint %s -r javascript/ola/' %
+    config['JS_LINT_ARGS'])
+js_lint_factory = BuildFactory()
+js_lint_factory.addStep(Git(repourl=("%s%s" % (config['BASEURL'], config['REPOSTUB']))))
+js_lint_factory.addStep(ShellCommand(
+   command=js_lint_command,
+   haltOnFailure=False,
+   flunkOnFailure=False,
+   warnOnWarnings=True,
+   warnOnFailure=True,
+   description='linting',
+   descriptionDone='lint',
+))
+
+# heap check factory
+hc_factory = BuildFactory()
+hc_factory.addStep(Git(repourl=("%s%s" % (config['BASEURL'], config['REPOSTUB']))))
+hc_factory.addStep(ShellCommand(
+    command=["qmake"],
+    name="qmake",
+    haltOnFailure=True,
+    flunkOnFailure=True,
+))
+hc_factory.addStep(Compile(
+    name='make',
+    description='building',
+    descriptionDone='build',
+    command=['make']))
+hc_factory.addStep(Compile(
+    name='unit tests',
+    description='testing',
+    descriptionDone='test',
+    command=['./unittest.sh'],
+    env={'HEAPCHECK': 'normal',
+         'PPROF_PATH': '/usr/bin/google-pprof'},
+    logfiles=config['LOGFILES'],
+    lazylogfiles=True,
+))
+
+# doxygen doc factory
+doc_factory = BuildFactory()
```

Comment:
This is now skipped due to having no configured slaves. They do have doxygen already.